### DISCUSSION TITLE
Welcome to Login Screen Discussions!

### DISCUSSION THREAD

**Discussion by dandln** [TheEpicSnowWolf/Foundry-VTT-Prettier-Login-Screen#11] [Announcements]
Creating this space here for anyone to ask questions about how to implement the CSS/use the module, as well as for suggestions for improvements! If you notice something that's not working as intended, please still open an [issue](https://github.com/TheEpicSnowWolf/Foundry-VTT-Prettier-Login-Screen/issues). Happy chatting :)

**Comment by XiaoKe89** [comment]
So I just updated to V10 and thought I'd trying using the module instead of manually updating the css like before and it seems to be working, only thing is I can't seem to get the header and footer images to work. I suspect either I'm not putting in the right path or I'm storing the images in the wrong location. I just put them in my foundry data folder so I put the path as follows into the parenthesis in the url field: (FoundryVTT/Data/worlds/my-world/Images/Header.png)
Is that how it should be put in? Or did I mistype something? Or do the images need to be somewhere else?

**Comment by xxUltiMoxx** [comment]
Two things come to mind here...
First, the path to your image should be in qoutes "pathToImage".
Second, foundry is self referencing, so try your PATH without the Foundryvtt/ at the beginning.

**Reply by XiaoKe89** [reply]
So I thought the quotes might be the issue so I tried that and it's still not working for me. I also tried both including and removing the Foundryvtt part. So this is what the line looks like now: "url("Data/worlds/my-world/Images/Header.png")"

**Reply by xxUltiMoxx** [reply]
![image](https://user-images.githubusercontent.com/22027445/208429928-372ef8ef-6c7e-4d13-96a5-688b88ff892d.png)

For me, the images are saved in the module folder, and as you can see, you dont even have to reference the data folder. 😉

**Reply by XiaoKe89** [reply]
I figured it out! I had the structure wrong. So using the structure I had it set to it should be: 
url("/worlds/my-world/Images/Header.png")

I just needed to leave out the "Foundryvtt" and the "Data" structure

Thanks for the help!

**Comment by kewnoe** [comment]
Hi! I tried to install from the link as I'm using Forge for hosting. Should it work? It lets me to activate the module, but after that I don't get the language option in settings.

I'm using Foundry version 10, build 291.

**Reply by dandln** [reply]
Hey! I've never used the Forge myself, but heard that it's quite restrictive in what you can and cannot customise. Is the option to change languages missing completely or can you just not select Latin? If the former then that's likely a restriction put in place by the Forge and nothing I can change sadly.

**Reply by kewnoe** [reply]
Hi! Thank you for answering. :) It has option "Latin" without any extra text. I have it there regardless of activating the module or not. I understood it should have extra text after it?
Here's an image: 

![image](https://user-images.githubusercontent.com/132590840/236631099-9456ebca-cbea-45b0-9ecf-ae7d6d05e060.png)

**Reply by dandln** [reply]
Sorry for the delayed reply! You need to select the translation in the Foundry settings themsleves (where you manage your worlds and modules etc, under "Configuration"), not in your world options, because the CSS will be loaded prior to the world itself. There it should also say the module name behind the language. :)

**Reply by kewnoe** [reply]
No worries! And thank you, I checked it and unfortunately there's no Latin -option in the configurations. So it doesn't work on Forge I believe. Sad, but I might switch to self-hosting in some point and then I can use your lovely module!

**Comment by Ging3rsnap** [comment]
Will we get a version of this for v11? That'd be wonderful if we could, because v11 breaks the image header and footer.

**Reply by dandln** [reply]
Most likely yes! I'm quite busy at the moment, but will hopefully get a chance to look into it over the next couple of weeks. I've not had a chance to upgrade/play around with v11 myself yet, but read about the new possibilites for login/setup screen modifications which are pretty exciting, so I'll have to see how this module will be able to natively integrate with that. But yeah, definitely plan on working on it!

**Comment by Routable** [comment]
I just wanted to say thanks for the work done here. 

I was about to build my own stylesheet from scratch, but stumbled on your Reddit post, and then this repo. People don't realize the effort something like this takes, and more importantly, to maintain, so I just wanted to pass on some appreciation 😊

**Reply by dandln** [reply]
That's really kind of you, thank you! :)
